LICENSE

```diff
@@ -0,0 +1,379 @@
+This software is copyright (c) 2011 by DuckDuckGo, Inc. http://duckduckgo.com,
```

Comment:
Hi @aanarres. Thanks for checking. I've just submitted a PR (https://github.com/duckduckgo/community-platform/pull/1374) for `dist.ini` so that your PR can focus on licensing.